<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
 background: qlineargradient(spread:pad, x1:0.498, y1:0.9265, x2:0.482587, y2:0.119, stop:0 rgb(170, 170, 170), stop:1 rgb(240, 240, 240));
}

/* Entradas de texto */
QLineEdit {
    background-color: #ffffff;
    border: 2px solid #c0c0c0;
    padding: 5px;
}

/* Texto */
QLabel {
    color: #000000;
    font-size: 14px;
}

/* Botón */
QPushButton {
    background-color: #4CAF50; /* Color de fondo */
    border: none;
    color: white;
    text-align: center;
    text-decoration: none;
    font-size: 14px;
    margin: 4px 2px;
    border-radius: 10px;
}

/*Botón está presionado */
QPushButton:pressed {
    background-color: #45a049;
}

/* Imagen */
QLabel#label_6 {
    border-image: url(:/src/vista/Imagenes/registro.png);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>121</y>
      <width>91</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Id Único</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>1º Apellido</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>2º Apellido</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>31</width>
      <height>442</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>211</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNombre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineApellido1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineApellido2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEmail">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>11</y>
      <width>31</width>
      <height>442</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="enviarBoton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(src/vista/Imagenes/registro.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
